core/src/test/java/apoc/export/csv/ImportCsvTest.java

```diff
@@ -430,6 +435,26 @@ public void testCustomRelationshipTypes() {
         Assert.assertEquals("John Jane", TestUtil.singleResultFirstColumn(db, "MATCH (p1:Person)-[:FRIENDS_WITH]->(p2:Person) RETURN p1.name + ' ' + p2.name AS pair ORDER BY pair"));
         Assert.assertEquals("Jane John", TestUtil.singleResultFirstColumn(db, "MATCH (p1:Person)-[:KNOWS]->(p2:Person) RETURN p1.name + ' ' + p2.name AS pair ORDER BY pair"));
     }
+
+    @Test
+    public void testEmptyDate() {
```

Comment:
I tried it locally, and this test is passing even without the fix. Would it be possible to tweak it to actually show the issue?

Reply:
Mmh this is really weird. I just tried this test without my fix and it fails (I also rebased it to be sure).
I really don't know why it doesn't fail you too 😅

Reply:
Never mind, I run the wrong test 🤦‍♀️